src/prefect/run_configs/vertex.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union, Iterable
+
+import yaml
+
+from prefect.run_configs.base import RunConfig
+from prefect.utilities.filesystems import parse_path
+
+
+class VertexRun(RunConfig):
+    """Configure a flow-run to run as an ECS Task.
```

Comment:
There's a pretty large surface area of additional arguments that can nominally be passed to vertex that we don't handle here explicitly. These are important ones IMO:

- network
- timeout (for restricting the duration of the job for safety)

I'd likely leave the following as out of scope:
- [pythonPackageSpec](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/CustomJobSpec#pythonpackagespec)
- additional [workerPoolSpecs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1/CustomJobSpec#workerpoolspec) or using more than one machine in the first workerPoolSpec

Defer to @gabcoyne if there would be any interactions there that we may want to support (wondering for example if there are interactions with non-docker storage or similar)

Reply:
As we talked about, lets make this is out of the box as possible with image and instance size as the only arguments that can be passed explicitly

Reply:
I would probably still include the GPU since that is pretty heavily used.